### ISSUE TITLE
ScrollView scrollToCoordinates not working as expected 

### ISSUE THREAD

**Issue by reid-mcpherson** [robolectric/robolectric#6176]
### Descriptions
ShadowView.scrollToCoordinates not working with ScrollView
### Steps to Reproduce

scrollView.scrollTo(0, 250)
assertThat(shadowOf(scrollView).scrollToCoordinates).isEqualTo(Point(0, 250))

expected: Point(0, 250)
but was : Point(0, 0)

### Robolectric & Android Version
Robolectric: 4.3.1
Android: 28

### Link to a public git repo demonstrating the problem:
https://github.com/reid-mcpherson/ScrollToCoordsBug

**Comment by utzcoz** [comment]
The robolectric doesn't support shadow `scrollTo` now.

**Comment by utzcoz** [comment]
Hi @hoisie, I have sent a pr to support `ScrollView.scrollTo` in https://github.com/robolectric/robolectric/pull/5668, and closed it after long time no-response. If you think it is needed, I can resend it again. But from the discussion of that pr, maybe the official recommends to use `ScrollView.smoothScrollTo` supported by Robolectric already.

**Comment by hoisie** [comment]
@utzcoz thanks for opening the PR, not sure why it slipped. off my radar. I could take a closer look at it.

**Comment by utzcoz** [comment]
@hoisie Maybe native graphics support `scrollTo` now?

**Comment by jeprubio** [comment]
Yup, it is, `scrollview.scrollTo()` is working now with native graphics. Thanks!
